### DISCUSSION TITLE
Onboarding PR have no labels

### DISCUSSION THREAD

**Discussion by ademar59** [renovatebot/renovate#13357] [Request Help]
### What would you like Renovate to be able to do?

I have set:

`"labels": ["dependencies"],`

And Onboarding pr have no labels.

### If you have any ideas on how this should be implemented, please tell us here.

A bug or a feature request to add a new config parameter like the undocumented `dependencyDashboardLabels` : `onboardingLabels`

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
Labels are applied to onboarding PRs at time of PR creation only. If you add `labels` to your bot config after onboarding PRs exist, they won't be added.

**Reply by ademar59** [reply]
I'm just recreating an onboarding and create a new PR (MR because gitlab)

![image](https://user-images.githubusercontent.com/8971839/148033466-5f3f35fe-47f8-406c-881d-575bbce1ebf2.png)

and no labels

the log:

```
INFO: Repository started (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts)
       "renovateVersion": "31.15.0"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts)
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts)
 INFO: Branch created (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts, branch=renovate/configure)
       "commit": "2d05d8f",
       "onboarding": true
 INFO: Dependency extraction complete (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts)
       "baseBranch": "master",
       "stats": {
         "managers": {
           "gitlabci": {"fileCount": 1, "depCount": 1},
           "gitlabci-include": {"fileCount": 1, "depCount": 1}
         },
         "total": {"fileCount": 2, "depCount": 2}
       }
 INFO: Onboarding PR created (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts)
       "pr": "Merge Request #23"
 INFO: Repository finished (repository=mobivia/driverservices/cloud-platform/digipaas-components/aiven/aiven-scripts)
       "durationMs": 11304
```

**Reply by rarkins** [reply]
Log at debug level

**Reply by ademar59** [reply]
Hi,

because i've seen the config in debug mode.
I try to add RENOVATE_LABELS=dependencies as env variable

But with or without, no labels

I join you the log as attachment
[renovate.-log.txt](https://github.com/renovatebot/renovate/files/7808875/renovate.-log.txt)

**Reply by rarkins** [reply]
Can you test the same thing against GitHub, to confirm that it's not something gitlab-specific?

**Reply by ademar59** [reply]
I unfortunately do not have a repo on github

**Reply by rarkins** [reply]
Github.com is free to use

**Comment by ademar59** [comment]
I've found in the code that labels is not used instead of addLabels who work fine

https://github.com/renovatebot/renovate/blob/b94c20bef4789321c18febb9213843e1566e22ec/lib/workers/repository/onboarding/pr/index.ts#L130

**Reply by rarkins** [reply]
I can't recall any reason for that to be intentional so unless anyone can remember one then we should fix it to include both

**Reply by HonkingGoose** [reply]
https://github.com/renovatebot/renovate/blob/b94c20bef4789321c18febb9213843e1566e22ec/lib/workers/repository/onboarding/pr/index.ts#L130

> I can't recall any reason for that to be intentional so unless anyone can remember one then we should fix it to include both

PR #9947 last changed this line of code.

**Reply by secustor** [reply]
No, I can't remember either.

**Reply by secustor** [reply]
I have created an issue to track this. https://github.com/renovatebot/renovate/issues/13431
